src/pages/collab/index.tsx

```diff
@@ -1,12 +1,16 @@
-import { useEffect, useRef, useState } from "react";
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
The signup link should be changed to sign-up here.
Not important tho, prob gonna nid one pr to impl a wrapper component for the auth.
Alrdy implemented one in #27, but not the best practice to keep it there

Reply:
Sure, I can remove the popup, it was just there for users to really confirm that they want to leave the page while queueing